File: 771_Jewels_and_Stones.js
var numJewelsInStones = function(jewels, stones) {
    let count = 0;
    let map = new Map();
    for(let i = 0; i < stones.length; i++){
        if(jewels.indexOf(stones[i]) != -1){
            count++;
        }
    }
    return count;
};

File: 771_Jewels_and_Stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        set<char> st;
        for(char c : jewels){
            st.insert(c);
        }
        int count = 0;
        for(char c : stones){
            if(st.find(c) != st.end()){
                count++;
            }
        }
        return count;
    }
};
